indigo_app/templates/indigo_api/work_overview.html

```diff
@@ -168,7 +168,6 @@ <h4>Active tasks</h4>
     </div>
   </div>
 
-  {% task_list place=place work=work active=True as active_tasks %}
   {% include 'indigo_api/_task_list.html' with tasks=active_tasks %}
```

Comment:
You may need to wrap this in `<div class="card">...</div>`? I removed the surrounding card from _task_list.html because I needed to put the filter form in the card header.

Reply:
done, thanks!